### ISSUE TITLE
lz4 dll installed with wrong filename

### ISSUE THREAD

**Issue by zmike** [vtorri/ewpi#12]
The installed file is `liblz4-1.dll` but it should be `liblz4.1.dll`. This causes runtime errors.

**Comment by vtorri** [comment]
indeed, i did something wrong. Anyway, i'll update ewpi with the latest lz4 release (1.9.0) and my fixes in upstream should solve the DLL name. Thank you

**Comment by vtorri** [comment]
if you have time, can you test latest git ? normally it is solved.
